srcStatic/IniModuleLoader.h

```diff
@@ -26,13 +26,15 @@ class IniModuleLoader
 	{
 		HINSTANCE handle;
 		DestroyModFunc destroyModFunc;
+		// The module's name is the same as the source ini file SectionName
+		std::string name;
```

Comment:
I would like to document somewhere that the moduleName is the same as the sectionName. It could be documented as a comment in the function GetModuleName if you prefer.

Reply:
Actually, yes, that should probably be documented for `GetModuleName`. That's the public interface where callers will care about what a `moduleName` actually is. In contrast, the `IniModuleEntry` is a private struct which the user shouldn't know or care about.